flutter/lib/screens/home_screen.dart

```diff
@@ -330,4 +180,123 @@ class _SettleHomePageState extends State<SettleHomePage> {
       },
     );
   }
+
+  Widget build(BuildContext context) {

```

Comment:
Can you refactor this method any more? I think it'd be cleaner to have some of this stuff defined in variables and then pass the variables in, like the drawer and buttons